src/Umbraco.Web.UI.Client/src/less/modals.less

```diff
@@ -111,7 +111,7 @@
 	left: 0px;
 	right: 0px;
 	bottom: 0px;
-    padding: 20px;
+    padding: 10px;
```

Comment:
Hence my reply from yesterday, shouldn't the padding be `8px` instead?

Reply:
@abjerner you're right. That however left behind a gap between the dialog body and footer (the scroll bar didn't align with the footer), but I believe I got that fixed as well.

Here's a zoomed in view of dialog buttons (left) and content editor buttons (right):

![image](https://user-images.githubusercontent.com/7405322/67641879-56081100-f906-11e9-9720-093283db6242.png)